### DISCUSSION TITLE
在dify工作流使用讯飞星火大模型的spark lite模型，运行时报错：[spark] Error: [Spark] code: 400, error: Code: 11200, Error: AppIdNoAuthError:(23:45:53.182) tokens.total，Help

### DISCUSSION THREAD

**Discussion by skystar-lwx** [langgenius/dify#9504] [Help] [answered]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### 1. Is this request related to a challenge you're experiencing? Tell me about your story.

在dify工作流使用讯飞星火大模型的spark lite模型，运行时报错：[spark] Error: [Spark] code: 400, error: Code: 11200, Error: AppIdNoAuthError:(23:45:53.182) tokens.total

### 2. Additional context or comments

![F9AD1863-21AE-4ff0-986C-48B11DF25F3D](https://github.com/user-attachments/assets/66fb1c11-8cc5-4441-bc1e-494c4fcb736f)

**Comment by chent199** [comment, marked as answer]
### 11200 | 授权错误：该appId没有相关功能的授权 或者 业务量超过限制
[讯飞文档链接] (https://www.xfyun.cn/doc/spark/%E6%8E%A5%E5%8F%A3%E8%AF%B4%E6%98%8E.html)

**Reply by TXECHYES** [reply]
解决了吗？请问如何解决的？

**Reply by tiaotiao97** [reply]
how to solve...?
